src/core/utils.ts

```diff
@@ -1,16 +1,3 @@
-/**
```

Comment:
why remove license?

Reply:
Arg, typescript bug where it removes the license comment if you apply the edit action to remove an unused import.